src/main/java/de/tum/in/www1/artemis/service/EntityFileHelper.java

```diff
@@ -10,24 +8,20 @@
 import javax.annotation.Nullable;
 
 import org.apache.commons.io.FileUtils;
-import org.jvnet.hk2.annotations.Service;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
-import org.springframework.context.annotation.Profile;
 
 /**
  * Service for handling file operations for entities.
  */
-@Profile(PROFILE_CORE)
-@Service
 @Deprecated(forRemoval = true)
-public class EntityFileService {
+public class EntityFileHelper {
 
-    private static final Logger log = LoggerFactory.getLogger(EntityFileService.class);
+    private static final Logger log = LoggerFactory.getLogger(EntityFileHelper.class);
 
     private final FileService fileService;
 
-    public EntityFileService(FileService fileService) {
+    public EntityFileHelper(FileService fileService) {
```

Comment:
This class is only constructed manually, right? Therefore the annotations are not needed for dependency injection?

I guess unit test results would show if there are failing usages with injection.

Reply:
yes, it is only constructed manually once and deprecated anyway. It will be removed soon